src/main/scala/org/zalando/react/nakadi/ReactiveNakadi.scala

```diff
@@ -86,6 +96,66 @@ class ReactiveNakadi {
     consumeWithOffsetSink(props, ReactiveNakadi.ConsumerDefaultDispatcher)
   }
 
+  def producerFlow(producerProperties: ProducerProperties)(implicit system: ActorSystem): Flow[ProducerMessage, Seq[String], NotUsed] = {
+    import org.zalando.react.nakadi.client.models.JsonOps._
+
+
+    import akka.http.scaladsl.model.MediaTypes._
+    import akka.http.scaladsl.model.headers._
+    import concurrent.duration._
+
+    implicit val materializer = ActorMaterializer()
```

Comment:
Maybe use this: `ActorMaterializer(ActorMaterializerSettings(context.system))` as i believe it will use the `ActorMaterializer` that is defined by the user of the library.